content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst

```diff
@@ -59,56 +65,106 @@ which is “Units” here.
 Specify a product's units of measure
 ====================================
 
-To set units of measure on a product, first go to :menuselection:`Inventory --> Products -->
-Products` and click on a product to open its settings. Then, click on :guilabel:`Edit`.
+To set units of measure on a product, first go to :menuselection:`Inventory app --> Products -->
+Products` and select a product to open its product form page.
 
 In the :guilabel:`General Information` tab, edit the :guilabel:`Unit of Measure` field to specify
-the unit of measure that the product is sold in. The specified unit will also be the unit used to
-keep track of the product's inventory and internal transfers. Edit the :guilabel:`Purchase UoM`
-field to specify the unit of measure that the product is purchased in.
+the unit of measure that the product is sold in. The specified unit is also the unit used to keep
+track of the product's inventory and internal transfers.
+
+Edit the :guilabel:`Purchase UoM` field to specify the unit of measure that the product is purchased
+in.
+
+.. _inventory/product_replenishment/unit-conversion:
 
 Unit conversion
 ===============
 
+Odoo automatically converts unit measurements when products have different :abbr:`UoMs (Units of
+Measure)` and purchase :abbr:`UoMs (Units of Measure)`.
+
+This occurs in various scenarios, including:
+
+#. :ref:`Vendor Orders <inventory/product_replenishment/buy-in-uom>`: purchase |UOM| on purchase
+   orders (PO) converts to |UOM| on internal warehouse documents
+#. :ref:`Automatic Replenishment <inventory/product_replenishment/replenish>`: generates |POs| when
+   the stock levels of a product (tracked in |UOM|) dips below a certain level. But, the |POs| are
+   created using the purchase |UOM|
+#. :ref:`Sell Products <inventory/product_replenishment/sell-in-uom>`: if a different |UOM| is used
+   on the sales order (SO), the quantity is converted to the warehouse's preferred |UOM| on the
+   delivery order
+
+.. _inventory/product_replenishment/buy-in-uom:
+
 Buy products in the Purchase UoM
 --------------------------------
 
-When creating a new request for quotation (RFQ) in the Purchase app, Odoo automatically uses the
-product's specified purchase unit of measure. However, if needed, the :guilabel:`UoM` can be
-manually edited on the RFQ.
+When creating a new request for quotation (RFQ) in the *Purchase* app, Odoo automatically uses the
+product's specified purchase unit of measure. If needed, the :guilabel:`UoM` can be manually edited
```

Comment:
As it's written this sentence is conditionally + indirectly referencing the **UoM** field, so to justify the gui, either:
- remove passive voice and turn into a more active instruction 
- actually say that it's a field on the form

```suggestion
product's specified purchase unit of measure. If needed, the :guilabel:`UoM` field value can be
```
OR
```suggestion
product's specified purchase unit of measure. If needed, manually edit the :guilabel:`UoM` value on
```